docs/data/base/components/popper/SimplePopper.js

```diff
@@ -1,5 +1,16 @@
 import * as React from 'react';
 import PopperUnstyled from '@mui/base/PopperUnstyled';
+import { styled } from '@mui/system';
+
+const StyledPopperDiv = styled('div')(
+  ({ theme }) => `
+  padding: 0.5rem;
+  border: 1px solid;
+  background-color: ${theme.palette.background.paper};
```

Comment:
Could we use something similar to the modal demos: https://github.com/mui/material-ui/blob/master/docs/data/base/components/modal/NestedModal.tsx#L50

This works at this moment because we use the Material UI structure theme on the docs. We don't want people of the base package to need to know this detail.

Reply:
I see your point. Makes sense.